server/templates/static/homepage.html

```diff
@@ -26,258 +26,314 @@
+          </li>
+          <li>
+            Or the progression in Europe over time (<a
+            href="/tools/map#&sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&dt=2040-06&pd=europe&pt=Continent&ept=IPCCPlace_50">2040</a>,
+            <a
+            href="/tools/map#&sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&dt=2045-06&pd=europe&pt=Continent&ept=IPCCPlace_50">2045</a>,
+            <a
+            href="/tools/map#&sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&dt=2050-06&pd=europe&pt=Continent&ept=IPCCPlace_50">2050</a>)
+          </li>
+        </ul>
+        <span class="clearfix"></span>
+      </p>
+      <p class="clearfix">
+        <span class="img-card float-md-left" style="width: 312px">
+          <a href="/tools/scatter#&svx=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&svy=Percent_Person_WithCoronaryHeartDisease&epd=country/USA&ept=County&dd=1">
+            <img class="img-thumbnail" src="/images/home-us_heart_disease_temp.png" />
+          </a>
+          <small class="caption">Heart condition vs. max projected summer temperature for US counties (RCP 4.5) (source: CDC, NASA)</small>
+        </span>
+        Climate change is not just about reducing carbon emissions. It is also
+        about adaptation to the change that is already happening. Note the
+        outliers in the upper right quadrant of this chart: counties like <a
+        href="{{ url_for('place.place', place_dcid='geoId/46121') }}">Todd County,
+        SD</a> and <a href="{{ url_for('place.place', place_dcid='geoId/46102')
+        }}">Oglala Lakota County, SD</a> with high incidence of coronary disease can
+        also expect some of the highest the highest temperature rises.
+      </p>
+    </article>
 
+    <article>
+      <h3>Emissions</h3>
+      <p>
+        <span class="img-card float-md-right" style="width: 312px">
+          <a href="/tools/map#&sv=Annual_Emissions_CarbonDioxide_NonBiogenic&pc=0&denom=Count_Person&pd=country/USA&pt=Country&ept=State&ppt=EpaReportingFacility">
+            <img class="img-thumbnail" src="/images/home-us_emissions.png" />
+          </a>
+          <small class="caption">Annual non-biogenic CO<sub>2</sub> emissions from large facilities across US states (source: EPA)</small>
+        </span>
+        Emissions are responsible for the climate crisis. If we look at <a
+        href="/tools/map#&sv=Annual_Emissions_CarbonDioxide_NonBiogenic&pc=0&denom=Count_Person&pd=country/USA&ept=State">large industrial emissions</a>, we see that Texas is the
+        highest emitter, at almost four times more than the next. In fact Texas
+        emits as much as both <a href="/tools/timeline#place=country/TUR,country/GBR,geoId/48&statsVar=Annual_Emissions_CarbonDioxide_NonBiogenic__eia/INTL.4008-8-MMTCD.A">Turkey and the UK</a>. In Texas,
```

Comment:
The timeline chart here links to two separate chart. I guess here needs to find the same stat var that applies to all three places?